src/components/cta/ExternalsAuth.tsx

```diff
@@ -51,13 +51,44 @@ const GitlabColorIcon: React.FunctionComponent<React.PropsWithChildren<{ classNa
     </svg>
 )
 
+const GoogleColorIcon: React.FunctionComponent<React.PropsWithChildren<{ className?: string }>> = ({ className }) => (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      xmlnsXlink="http://www.w3.org/1999/xlink"
+      width="46"
+      height="46"
+      viewBox="0 0 46 46"
+    >
```

Comment:
You need to pass the class name through here, to fix the warning and so your `pr-4` class where this is used below lands on a DOM element.

```suggestion
      className={className}
    >
```